src/main/App.ts

```diff
@@ -53,10 +53,16 @@ export class App {
 
       const redisClient = await this.createRedisClient();
       const firebaseAdmin = await initFirebaseAdmin();
+
       const geocoder = nodeGeocoder({
         provider: 'locationiq',
         apiKey: process.env.GEOCODING_KEY,
       });
+      const countryCode = 'ca';
+      const geocoderConfig: GeocoderConfig = {
+        geocoder: geocoder,
+        countryCode: countryCode,
+      };
```

Comment:
Do you think it is ok to hard code `ca` for `countryCode` variable? I mean, for now, every time we change the country code, other developer have to find this area. If we won't change the value in the future, that's fine!

Reply:
Ooo that's an excellent point! I'm not too sure if we'll change this variable in the future, but I can put a comment above this line or in the function itself to denote the default is currently Canada?

Reply:
Does country code need to be 'Ca'?

Reply:
I think we're focusing on Canada for now? @andrewzulaybar could you clarify? :)

Reply:
Yeah just Canada for now. @yunishimura0716 that's a good point, and we definitely want our geoCoder class to support multiple country codes. I think this is something to definitely think about when we make our app work outside of Canada.

Reply:
Yeah Canada for now!